staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/conversion.go

```diff
@@ -52,3 +53,20 @@ func Convert_v1alpha1_ObjectReference_To_audit_ObjectReference(in *ObjectReferen
 	}
 	return nil
 }
+
+func Convert_v1alpha1_Event_To_audit_Event(in *Event, out *audit.Event, s conversion.Scope) error {
+	if err := autoConvert_v1alpha1_Event_To_audit_Event(in, out, s); err != nil {
```

Comment:
if you get an v1alpha1 event with ObjectMeta.CreationTimestamp set, but StageTimestamp empty, what do you do? I would expect to copy CreationTimestamp in that case.

Same for Timestamp and RequestReceivedTimestamp.

Reply:
Done.